tools/gcp/comp_test/create_comptest_images.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
I am not a big fan of the "comp_test" name - without knowing the context, it's totally unclear what they are good for. Btw, "comp" could stand for "comparison" as well as "compatibility" tests.

In my mind, these tests are "backward compatility interop tests", "cross-version interop tests", "cross-release interop tests" or something in that sense.